webpack/ForemanRhCloudFills.js

```diff
@@ -31,7 +32,7 @@ const fills = [
   {
     slot: 'host-details-page-tabs',
     name: 'CVEs',
-    component: props => <CVEsHostDetailsTab {...props} />,
+    component: props => <CVEsHostDetailsTabWrapper {...props} />,
     weight: 300,
     metadata: {
       hideTab: vulnerabilityDisabled,
```

Comment:
What are the conditions for hiding the Vulnerabilities tab?

Reply:
The `vulnerabilityDisabled` [function](https://github.com/theforeman/foreman_rh_cloud/commit/a5e0be9d481d660c481bcbfdbb4131e4b67b5fd0#diff-665fd73109a9c7e08d48aabc70cebd24a9e398cdd81104a82d533006300b2a61R15-R16) determines whether the vulnerability feature should be considered disabled, either because:
* the host is not running RHEL, or
* the feature is not explicitly enabled on the host, which depends on whether the Red Hat Insights Advisor engine is active locally.

Reply:
I've got some issues with it when it got hidden, eventough I had the feature enabled. Could it be an issue with the RHEL check?

Reply:
Not sure, it's working fine for me. Maybe try debugging and see what's going on?